doc/partitioned-db.md

```diff
@@ -0,0 +1,206 @@
+# Partitioned Database
+
+When creating a Cloudant database, you can configure it as non-partitioned or partitioned.
+A partitioned database offers significant performance and cost advantages but requires you to specify a logical partitioning of your data.
+
+Reference:
```

Comment:
maybe this can be moved to the bottom of this section?  because assume users know about the details in cloudant and then come here, so i think it's more important to show them the connector specific info first.